php/sync/class-upload-sync.php

```diff
@@ -91,7 +90,8 @@ private function register_hooks() {
 			'bulk_actions-upload',
 			function ( $actions ) {
 				$cloudinary_actions = array(
-					'cloudinary-push' => __( 'Sync with Cloudinary', 'cloudinary' ),
+					'cloudinary-push'     => __( 'Sync with Cloudinary', 'cloudinary' ),
+					'cloudinary-clean-up' => __( 'Clean Cloudinary error', 'cloudinary' ),
```

Comment:
<!-- trivial preference-based requests -->
**🥜 nitpick**
This is so small but I think "error" should be capitalised to keep it consistent?

Reply:
```suggestion
					'cloudinary-clean-up' => __( 'Clean Cloudinary Error', 'cloudinary' ),
```